src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CallbackValidations.java

```diff
@@ -283,6 +284,42 @@ static boolean hasOcr(CaseDetails theCase) {
         return valid(classification);
     }
 
+    public static Validation<String, Classification> hasJourneyClassificationForAttachToCase(CaseDetails theCase) {
```

Comment:
Do we do anything with validation errors from this method?

Reply:
Will fix